dev/wlp-jakartaee-transform/rules/jakarta-xml-master.properties

```diff
@@ -72,6 +72,12 @@ faces-config1.xml=jakarta-faces-config.properties
 com.ibm.ws.security.inflow.context.provider.xml=jakarta-connectors.properties
 com.ibm.ws.jca17.processor.service.ConnectionFactoryResourceBuilder.xml=jakarta-resource.properties
 com.ibm.ws.security.javaeesec.cdi.extensions.JavaEESecCDIExtension.xml=jakarta-xml-securityEnterprise.properties
+#messaging
```

Comment:
See line 49, above.  That's where the `#Messaging` section starts.  We have a minor problem here.  Please... 
1) Change the right-hand-side of the DelayClient properties (lines 79,80) to `jakarta-metatype.properties`
2) Move the DelayClient properties to immediately after line 54 (just before the `# Metatype` comment)
3) Delete the comment starting at `#messaging`. 
4) Delete the jakarta-messaging-metatype.properties iff the file is now vestigial.

Reply:
Updated.  See the new commit for the changes.  I'm leaving both properties files, for now.